crates/arrow/src/datatypes/datatype.rs

```diff
@@ -146,6 +148,19 @@ pub enum DataType {
     Decimal(usize, usize),
 }
 
+/// The time zone is a string indicating the name of a time zone, one of:
+///
+/// * As used in the Olson time zone database (the "tz database" or
+///   "tzdata"), such as "America/New_York"
+/// * An absolute time zone offset of the form +XX:XX or -XX:XX, such as +07:30
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct TimeZone {
```

Comment:
move to base crate, rename like BaseTimeZone (different to general chrono::TimeZone)  and then make arrow depending on base, we let base as a primtive building blocks for arrow rather than reversed.